config/chance_weight.cwt

```diff
@@ -84,45 +71,87 @@ single_alias[weight_chance_modifier] = {
     ## cardinality = 0..1
     desc = localisation
 }
-
-single_alias[opinion_chance_modifier] = {
+# TODO: Separate these out into target or opinion target
+alias[weight:opinion_modifier] = {
```

Comment:
Calling this opinion modifier is somewhat dicey since it could be confused with opinion_modifiers, which of course are similar but distinct from character_modifiers.

Yay Jomini!